doc/api/http.md

```diff
@@ -919,6 +919,8 @@ timeout has fired, it will reset the regular inactivity timeout, i.e.,
 [`server.timeout`][].
 
 A value of 0 will disable the keep-alive timeout behavior on incoming connections.
+A value of 0 makes the http server behaves similarly to earlier nodejs versions,
```

Comment:
Please change `behaves` to `behave` and `nodejs` to `Node.js`.

Reply:
Also, while we're in here, it would be good to put backticks around `0` so it's ``` `0` ```?